src/abstract-ops/function-operations.mts

```diff
@@ -531,7 +531,7 @@ function BuiltinFunctionConstruct(this: BuiltinFunctionObject, argumentsList: Ar
 }
 
 /** https://tc39.es/ecma262/#sec-createbuiltinfunction */
-export function CreateBuiltinFunction(steps, length: number, name: PropertyKeyValue | PrivateName, internalSlotsList: readonly string[], realm?: Realm, prototype?: ObjectValue | NullValue, prefix?: string, isConstructor = Value.false): FunctionObject {
+export function CreateBuiltinFunction(steps: (values: readonly Value[], context: { thisValue: ObjectValue | UndefinedValue, NewTarget: ObjectValue | UndefinedValue }) => Completion | Value, length: number, name: PropertyKeyValue | PrivateName, internalSlotsList: readonly string[], realm?: Realm, prototype?: ObjectValue | NullValue, prefix?: string, isConstructor = Value.false): FunctionObject {
```

Comment:
This could’ve used the `Arguments` type import from `../value.mts`:
```suggestion
export function CreateBuiltinFunction(steps: (args: Arguments, context: { thisValue: ObjectValue | UndefinedValue, NewTarget: ObjectValue | UndefinedValue }) => Completion | Value, length: number, name: PropertyKeyValue | PrivateName, internalSlotsList: readonly string[], realm?: Realm, prototype?: ObjectValue | NullValue, prefix?: string, isConstructor = Value.false): FunctionObject {
```

Reply:
thanks! will fix on my next pr